config/locales/en/user_mailer.yml

```
@@ -11,7 +11,7 @@ en:
 
         If you have a DfE Sign-in account, you can use it to sign in:
 
-        [link](%{sign_in_url})
+        %{sign_in_url}
```

This will return plaintext. I think you'll want to do:

```suggestion
        [sign_in_url](%{sign_in_url})
```